cmd/bnsd/x/qualityscore/codec.proto

```diff
@@ -11,14 +11,14 @@ message Configuration {
   // This defines the Address that is allowed to update the Configuration object and is
   // needed to make use of gconf.NewUpdateConfigurationHandler
   bytes owner = 2 [(gogoproto.casttype) = "github.com/iov-one/weave.Address"];
-  float c = 3;
-  float k = 4;
-  float kp = 5;
-  float q0 = 6;
-  float x = 7;
-  float x_inf = 8;
-  float x_sup = 9;
-  float delta = 10;
+  weave.Fraction c = 3 [(gogoproto.nullable) = false];
```

Comment:
Does `nullable = false` mean that the parameters cannot be removed entirely from the configuration?

Reply:
No, this is a go specific setting. It means that the generated Go code will not generate a pointer but embed structure. This way `nil` value cannot be assigned and a value is always present. This rule is Go only and it is irrelevant for the protobuf.